### PR TITLE
Remove stale comments

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Fixes https://github.com/dotnet/roslyn/issues/67329

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
@jcouv  ptal.

File: src/Compilers/CSharp/Test/Symbol/Compilation/UsedAssembliesTests.cs [hunk 1]
```diff
@@ -3732,7 +3732,6 @@ public unsafe static void Main()
 }
 ");
 
-            // Support for `using unsafe static` tracked by https://github.com/dotnet/roslyn/issues/67329
```
Comment (jcouv) [inline src/Compilers/CSharp/Test/Symbol/Compilation/UsedAssembliesTests.cs:3735]: The comment was about the `using static C1<S<C0>*[]>;` scenario. Do we have the corresponding test scenario with "unsafe" added?
Same question for `using static C1<S<C0>*[]>.E1;` below.

**Comment by CyrusNajmabadi** [comment]
tests added @jcouv .

**Review by jcouv — APPROVED** [review]
LGTM (iteration 2). Danke schön!

**Pull request merged**